cypress/integration/banner.spec.ts

```diff
@@ -4,31 +4,79 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import { banner } from '../support/config'
+const BANNER_LOCAL_STORAGE_KEY = 'banner.etag'
+const MOCK_E_TAG = 'mockETag'
+const bannerMocKContent = 'This is the mock banner call'
 
 describe('Banner', () => {
   beforeEach(() => {
+    cy.intercept({
+      method: 'GET',
+      url: '/public/banner.txt'
+    }, {
+      statusCode: 200,
+      headers: { 'ETag': MOCK_E_TAG },
+      body: bannerMocKContent
+    })
+      .as('getBanner')
+    cy.intercept({
+      method: 'HEAD',
+      url: '/public/banner.txt'
+    }, {
+      statusCode: 200,
+      headers: { 'ETag': MOCK_E_TAG }
+    })
+      .as('headBanner')
+
     cy.visit('/')
-    expect(localStorage.getItem('bannerTimeStamp')).to.be.null
+    localStorage.removeItem(BANNER_LOCAL_STORAGE_KEY)
+    expect(localStorage.getItem(BANNER_LOCAL_STORAGE_KEY)).to.be.null
   })
 
   it('shows the correct alert banner text', () => {
-    cy.get('.alert-primary.show')
-      .contains(banner.text)
+    cy.get('[data-cy="motd-banner"]')
+      .contains(bannerMocKContent)
   })
 
   it('can be dismissed', () => {
-    cy.get('.alert-primary.show')
-      .contains(banner.text)
-    cy.get('.alert-primary.show')
-      .find('.fa-times')
+    cy.get('[data-cy="motd-banner"]')
+      .contains(bannerMocKContent)
+    cy.get('button[data-cy="motd-dismiss"]')
       .click()
       .then(() => {
-        expect(localStorage.getItem('bannerTimeStamp'))
+        expect(localStorage.getItem('banner.etag'))
           .to
-          .equal(banner.timestamp)
+          .be
```

Comment:
Why do you add the `.be`?

If you're aiming for "valid sentences", it needs to be "expect banner.etag to be equal to mock_e_tag". This is obviously not possible, therefore the usage of "equal" as a verb sounds better: "expect banner.etag to equal mock_e_tag".

Reply:
This was already there in the original code.